roles/openshift_openstack/templates/heat_stack.yaml.j2

```diff
@@ -15,9 +15,11 @@ outputs:
     description: IPs of the etcds
     value: { get_attr: [ etcd, private_ip ] }
 
+  {% if openshift_openstack_etcd_floating_ip | default(True) | bool %}
```

Comment:
All of these should be indented all the way to the left (i.e. no leading whitespace).

Otherwise the whitespace will be left in there after yaml is processed and that can cause issues.